src/link/MachO/Object.zig

```diff
@@ -579,6 +579,7 @@ pub fn parseDwarfInfo(self: Object) error{Overflow}!dwarf.DwarfInfo {
         .debug_info = &[0]u8{},
         .debug_abbrev = &[0]u8{},
         .debug_str = &[0]u8{},
+        .debug_str_offsets = null,
```

Comment:
Same comment as for COFF above

Reply:
I agree with @topolarity here - in fact, I'd just go full on fatal error here with an instructive comment such as

```suggestion
        .debug_str_offsets = unreachable, // TODO implement string offsets for DWARFv5
```

Reply:
I've not used that pattern before. Would that not trigger a panic unconditionally whenever `parseDwarfInfo` is called?

Reply:
Oh oops, you're absolutely right. Please ignore this suggestion and follow what @topolarity is suggesting :-)